CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+## Testing
+
+A full guide to testing can be found in the [testing](./TESTING.md) document
+
+## Releases
+
+To start a release:
+
+- decide on a version number
+- checkout the `next` branch
+- review commits made to `next` since the last release
+- update `CHANGELOG.md` reflecting the above changes, release version, and release date. Commit to `next`
```

Comment:
I would make this commit on the release branch

Reply:
Ah, yes, that is how it should be. Fixed.